library-backend/library-app/pom.xml

```diff
@@ -31,6 +31,21 @@
             <artifactId>library-impl</artifactId>
             <version>1.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
```

Comment:
I think we can remove this since the second one, springboot related, will also bring in this one.